Router.svelte

```diff
@@ -85,6 +96,14 @@ export const querystring = derived(
     ($loc) => $loc.querystring
 )
 
+/**
+ * Readable store that returns the current sub hash
+ */
+export const hash = derived(
```

Comment:
Good idea to export the sub-hash, but I would name it differently, such as `anchor`?